backend/game_flow/game_flow_websocket.py

```diff
@@ -75,25 +83,59 @@ def end_game(game_id):
     del games[game_id]
 
 
+def remove_player(game_id, player_id):
+    connections = player_connections_by_game_id[game_id]
+    print(game_id)
+    if player_id in connections:
+        del connections[player_id]
```

Comment:
just in case we can call .close() on it before removing it

Reply:
True yeah, will do